lib/Models/ResultPendingCatalogItem.ts

```diff
@@ -5,6 +5,9 @@ import ResultPendingCatalogItemTraits from "../Traits/ResultPendingCatalogItemTr
 export default class ResultPendingCatalogItem extends CatalogMemberMixin(
   CreateModel(ResultPendingCatalogItemTraits)
 ) {
+  readonly showsInfo = true;
+  readonly isMappable = true;
```

Comment:
I think you are right about `isMappable`, we don't need to set it. 

We need `showsInfo` because in this case we are showing `parameters` for the request we are making and an also the error in case it fails. But I do think we can make it false by default and just turn it on if needed.

Reply:
![image](https://user-images.githubusercontent.com/1430/61921185-91114600-af9f-11e9-932c-3d9dfe8ce465.png)

This is the info panel for pending item from master that I am trying to recreate. It requires `isMappable` to be `true` due to the [check here](https://github.com/TerriaJS/terriajs/blob/mobx/lib/ReactViews/Preview/DataPreview.jsx#L43).